src/webgpu/api/validation/encoding/queries/begin_end.spec.ts

```diff
@@ -18,6 +18,12 @@ import { pbool } from '../../../../../common/framework/params_builder.js';
 import { makeTestGroup } from '../../../../../common/framework/test_group.js';
```

Comment:
nit: could you clarify that the TODO at the top is for pipeline statistics? E.g.

```diff
-TODO:
+TODO: for pipeline statistics queries:
```